src/data/markdown/docs/02 javascript api/07 k6-experimental/07 grpc.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "grpc"
+excerpt: "Experimental GRPC module"
+---
+
+The `k6/experimental/grpc` module is an extension of the [`k6/net/grpc`](/javascript-api/k6-net-grpc/). It provides a [gRPC](https://grpc.io/) client for Remote Procedure Calls (RPC) over HTTP/2. 
+
+The key-difference between the two modules is new `Stream` class, which provides client and server streaming support. Our long-term goal is to make this module part of k6 core, and long-term to replace the [`k6/net/grpc`](/javascript-api/k6-net-grpc/) module.
+
+| Class/Method | Description |
+|--------------|-------------|
+| [Client](/javascript-api/k6-experimental/grpc/client) | gRPC client used for making RPC calls to a gRPC Server. |
```

Comment:
I believe we shouldn't. The rationale is that since the code is separated, the docs should also be separated since, at some point, even for the client, some differentiation could appear.